tabpy-tools/tabpy_tools/client.py

```diff
@@ -20,10 +20,9 @@
 
 import os as _os
 
-
 import logging as _logging
```

Comment:
Any reason for private namespacing here?

Reply:
You are right - we don't need this. I am removing all private namespaces in the file.